src/components/Header/Tools.test.tsx

```
@@ -395,6 +395,80 @@ describe('Tools - toolbarConfig visibility', () => {
+      settingsGroups: {
+        hidePreview: true,
+        hideSettingsGroup: true,
+        hideIAM: true,
+        hideTheme: true,
+        hideContrastMode: true,
+      },
+    };
+    renderTools(
+      { 'platform.chrome.dark-mode': true, 'platform.chrome.high-contrast': true, 'platform.chrome.felt-theme': true },
+      lightwellConfig
+    );
```

ESLint

[error] Replace `⏎······{·'platform.chrome.dark-mode':·true,·'platform.chrome.high-contrast':·true,·'platform.chrome.felt-theme':·true·},⏎······lightwellConfig⏎····` with `{·'platform.chrome.dark-mode':·true,·'platform.chrome.high-contrast':·true,·'platform.chrome.felt-theme':·true·},·lightwellConfig`

(prettier/prettier)